tests/functional/spec/components/dropdown.spec.js

```diff
@@ -0,0 +1,138 @@
+    });
+
+    it('When I have not selected a dropdown option and click Continue, Then the overridden error message should be displayed', function() {
+      return browser
+        .click(DropdownMandatoryPage.submit())
+        .getText(DropdownMandatoryPage.errorNumber(1)).should.eventually.contain("Overridden test error message.");
+    });
+  });
+
+  //Optional
+  describe('Given I start a Optional Dropdown survey', function() {
+
+    const DropdownOptionalPage = require('../../pages/components/dropdown/optional/dropdown-optional.page');
+    const DropdownSummaryPage = require('../../pages/components/dropdown/optional/summary.page');
+    const schema = 'test_dropdown_optional.json';
+
+    it('When I have not selected a dropdown option, Then the summary should display "No answer provided"', function() {
+      return helpers.openQuestionnaire(schema)
+        .then(() => {
+          return browser
+            .click(DropdownOptionalPage.submit())
+            .getText(DropdownSummaryPage.dropdownOptionalAnswer()).should.eventually.contain("No answer provided");
+        });
+    });
+
+    it('When I have selected a dropdown option, Then the selected option should be displayed in the summary', function() {
+      return helpers.openQuestionnaire(schema)
+        .then(() => {
+          return browser
+            .selectByValue(DropdownOptionalPage.answer(), "Rugby is better!")
+            .click(DropdownOptionalPage.submit())
+            .getText(DropdownSummaryPage.dropdownOptionalAnswer()).should.eventually.contain("Rugby is better!");
+        });
+    });
+
+    it('When I have selected a dropdown option and I reselect the default option (Please select an answer etc), Then the summary should display "No answer provided"', function() {
+      return helpers.openQuestionnaire(schema)
+        .then(() => {
+          return browser
+            .selectByValue(DropdownOptionalPage.answer(), "Chelsea")
+            .selectByValue(DropdownOptionalPage.answer(), "")
+            .click(DropdownOptionalPage.submit())
+            .getText(DropdownSummaryPage.dropdownOptionalAnswer()).should.eventually.contain("No answer provided");
+        });
+    });
+
+    it('When I\'m on the summary page and I click Edit and change my answer, Then the newly selected option should be displayed in the summary', function() {
+      return helpers.openQuestionnaire(schema)
+        .then(() => {
+          return browser
+            .selectByValue(DropdownOptionalPage.answer(), "Rugby is better!")
+            .click(DropdownOptionalPage.submit())
+            .getText(DropdownSummaryPage.dropdownOptionalAnswer()).should.eventually.contain("Rugby is better!")
+            .click(DropdownSummaryPage.dropdownOptionalAnswerEdit())
+            .click(DropdownOptionalPage.submit())
+            .getText(DropdownSummaryPage.dropdownOptionalAnswer()).should.eventually.contain("Rugby is better!")
```

Comment:
This scenario is tested in the mandatory scenarios, I don't think we need to duplicate it for the optional dropdown.